lib/jsdom/living/helpers/iterable-weak-set.js

```diff
@@ -0,0 +1,56 @@
+"use strict";
+
+function cleanup({ ref, set }) {
```

Comment:
I recall it being important for these not to be closures for... reasons. Could you add a comment documenting these reasons?

Reply:
I believe that the reason is memory usage, since the `cleanup` static method can be shared between all instances.